Files/Views/SettingsPages/FilesAndFolders.xaml

```diff
@@ -46,7 +46,7 @@
 
             <local:SettingsBlockControl Title="{helpers:ResourceString Name=SettingsFilesAndFoldersHideSystemItems}" HorizontalAlignment="Stretch">
                 <local:SettingsBlockControl.Icon>
-                    <FontIcon Glyph="&#xED1A;" />
+                    <FontIcon Glyph="&#xE72E;" />
```

Comment:
I think we should use a path instead as the icon currently used is working on Windows 11.

Reply:
Would conditional xaml work here? Would look different on w10 and 11 though.

Reply:
> Would conditional xaml work here? Would look different on w10 and 11 though.

I had the same idea and currently I'm checking if we can use it!

Reply:
Yep, it seems that there is not problem on using it!